br: br debug encode panic

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

<!-- a step by step guide for reproducing the bug. -->
- br backup full
- br debug decode
- br debug encode
- panic

### 2. What did you expect to see? (Required)
Successfully encode and generate file `backupmeta_from_json` in target bucket.

### 3. What did you see instead (Required)
panic
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x2bf863b]

goroutine 1 [running]:
github.com/pingcap/tidb/br/pkg/utils.fromJSONSchema(0xc000fb4990)
        /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/pkg/utils/json.go:118 +0x3b
github.com/pingcap/tidb/br/pkg/utils.fromJSONBackupMeta(0xc000dbea80)
        /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/pkg/utils/json.go:167 +0x88
github.com/pingcap/tidb/br/pkg/utils.UnmarshalBackupMeta({0xc0010ca000, 0x3200b, 0x36000})
        /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/pkg/utils/json.go:28 +0x78
main.encodeBackupMetaCommand.func1(0xc0003bd900?, {0xc000a106c0?, 0x4?, 0x4?})
        /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/cmd/br/debug.go:342 +0x132
github.com/spf13/cobra.(*Command).execute(0xc0003bd900, {0xc00004e0a0, 0x4, 0x4})
        /go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:856 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0xc0003bca00)
        /go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
main.main()
        /home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/cmd/br/main.go:57 +0x325
```

### 4. What is your TiDB version? (Required)

<!-- Paste the output of SELECT tidb_version() -->
v6.1.0

--- a/br/pkg/utils/json.go
+++ b/br/pkg/utils/json.go
@@ -114,6 +114,10 @@ func makeJSONSchema(schema *backuppb.Schema) (*jsonSchema, error) {
 
 func fromJSONSchema(jSchema *jsonSchema) (*backuppb.Schema, error) {
 	schema := jSchema.Schema
+	if schema == nil {
+		schema = &backuppb.Schema{}
+	}
+
 	var err error
 	schema.Db, err = json.Marshal(jSchema.DB)
 	if err != nil {

--- a/br/pkg/utils/json_test.go
+++ b/br/pkg/utils/json_test.go
@@ -204,6 +204,44 @@ var testMetaJSONs = [][]byte{
   "is_raw_kv": true,
   "br_version": "BR\nRelease Version: v5.0.0-master\nGit Commit Hash: c0d60dae4998cf9ac40f02e5444731c15f0b2522\nGit Branch: HEAD\nGo Version: go1.13.4\nUTC Build Time: 2021-03-25 08:10:08\nRace Enabled: false"
 }`),
+	[]byte(`{
+    "files": [
+      {
+        "sha256": "3ae857ef9b379d498ae913434f1d47c3e90a55f3a4cd9074950bfbd163d5e5fc",
+        "start_key": "7480000000000000115f720000000000000000",
+        "end_key": "7480000000000000115f72ffffffffffffffff00",
+        "name": "1_20_9_36adb8cedcd7af34708edff520499e712e2cfdcb202f5707dc9305a031d55a98_1675066275424_write.sst",
+        "end_version": 439108573623222300,
+        "crc64xor": 16261462091570213000,
+        "total_kvs": 15,
+        "total_bytes": 1679,
+        "cf": "write",
+        "size": 2514,
+        "cipher_iv": "56MTbxA4CaNILpirKnBxUw=="
+      }
+    ],
+    "schemas": [
+      {
+        "db": {
+          "charset": "utf8mb4",
+          "collate": "utf8mb4_bin",
+          "db_name": {
+            "L": "test",
+            "O": "test"
+          },
+          "id": 1,
+          "policy_ref_info": null,
+          "state": 5
+        }
+      }
+    ],
+    "ddls": [],
+    "cluster_id": 7194351714070942000,
+    "cluster_version": "\"6.1.0\"\n",
+    "br_version": "BR\nRelease Version: v6.1.0\nGit Commit Hash: 1a89decdb192cbdce6a7b0020d71128bc964d30f\nGit Branch: heads/refs/tags/v6.1.0\nGo Version: go1.18.2\nUTC Build Time: 2022-06-05 05:09:12\nRace Enabled: false",
+    "end_version": 439108573623222300,
+    "new_collations_enabled": "True"
+  }`),
 }
 
 func TestEncodeAndDecode(t *testing.T) {

